smart-contracts/test/test_security.js

```diff
@@ -70,10 +73,31 @@ describe("Security Test", function () {
         owner,
         userOne,
         userThree,
-        pauser.address
+        pauser.address,
+        networkDescriptor
       );
     });
 
+    it("should allow operator to call reinitalize after initialization", async function () {
+      await expect(state.bridgeBank.connect(operator).reinitialize(
+        operator.address,
+        state.cosmosBridge.address,
+        owner.address,
+        pauser.address,
+        state.networkDescriptor
```

Comment:
Here let's also assert that the state changed to the expected variables with some expect statements.